org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/converter/ZigBeeBaseChannelConverter.java

```diff
@@ -306,11 +312,42 @@ public int getPollingPeriod() {
         return pollingPeriod;
     }
 
+    /**
+     * Gets the minimum of all maximum reporting periods used in attribute reports. If no attribute reports are
+     * configured, the default {@code Integer.MAX_VALUE} will be returned.
+     *
+     * @return minimum of all maximum reporting periods in seconds
+     */
+    public int getMinimalMaximumReportingPeriod() {
```

Comment:
This is  a horrible name - can't it just be ```getMinimumReportingPeriod```

Reply:
I thought about this too, BUT...

There is a minimum reporting period and a maximum reporting period for **each** attribute reporting. And what this variable should represent is the minimum of all those maximum reporting times :)

So I find `getMinimumReportingPeriod` confusing as well, because you could think of this as the minimum reporting period for an attribute, whereas i am only considering the maximums...

Reply:
I understand that, but for the sake of clarity I think it's much clearer to use the shorter name. I don't think it really needs to reflect exactly what is happening in the converter and you can always describe this in the text.